tests/libtest.sh

```diff
@@ -546,6 +546,29 @@ skip_without_user_xattrs () {
     fi
 }
 
+# Skip unless SELinux is disabled, or we can relabel.
+# Default Docker has security.selinux xattrs, but returns
+# EOPNOTSUPP when trying to set them, even to the existing value.
+# https://github.com/ostreedev/ostree/pull/1182
```

Comment:
Is this the right link? I don't see anything about that there.

Reply:
Fixup ⬇️